src/components/CryptoList.css

```diff
@@ -1,16 +1,31 @@
 .crypto-list {
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
   align-items: center;
+  justify-content: space-around;
+  font-size: 1em;
+  margin: 0;
+  padding: 0;
 }
 
 .crypto-card {
+  flex: 1;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 1.5em;
-  min-width: 14em;
+  padding: 1.5em 0;
+  min-width: 12em;
+  min-height: 10em;
+  font-size: 0.8em;
+}
+
+.crypto-card h3 {
+  font-size: 1em;
+}
+
+.crypto-card p {
+  font-size: 0.8em;
+  margin: 0.7em 0;
 }
 
 .crypto-card:nth-child(even) {
```

Comment:
- [ ] [OPTIONAL]
Good job, cards are displaying correctly on a mobile screen. However, it will be a good idea if the  styling of the list on your _Homepage_ conforms to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that the cards of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (or slightly alternating color shade) background color(s)**.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/Kaiserabbas/crypto-currencies/assets/12745474/b0debf00-0c0a-4fbd-838a-63e210af477b"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png">

Reply:
Thank you for the design suggestions, I will definitely work on this to improve user experience.